plugins/inputs/win_perf_counters/README.md

```diff
@@ -19,7 +19,19 @@ as counters used when performance monitoring
  There are a lot of other good objects to monitor, if you know what to look for.
  This file is likely to be updated in the future with more examples for
  useful configurations for separate scenarios.
-
+ 
+ ### Schema
+ 
+ *Measurement name* is specified per performance object or `win_perf_counters` by default.
+ 
+ *Tags:*
+  - computer - computer name, if specified in the `Computers` parameter, otherwise omitted
```

Comment:
I wonder if this should alwasys be set and be renamed to `source` as discussed in #4413

Reply:
`source` is very overloaded name. In this case, `computer` is more suitable as it is officially part of complete counter path, see [https://docs.microsoft.com/en-us/windows/win32/perfctrs/specifying-a-counter-path](https://docs.microsoft.com/en-us/windows/win32/perfctrs/specifying-a-counter-path)

Reply:
I don't disagree with your point but all the same this should be changed to `source` for forward consistency and special handling with some outputs.

Reply:
Ok